<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="twCommand">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>выполнить</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>команда</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>код</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ответ</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Контроль свободности/занятости участков пути в результате циклического опроса состояния</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twControl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>811</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Объект</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Принятое состояние</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>код</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>379</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер цикла</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_counter">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>380</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
